src/lakefs_spec/util.py

```diff
@@ -1,7 +1,9 @@
 import re
 
 from lakefs_sdk import __version__ as __lakefs_sdk_version__
-from lakefs_sdk.client import LakeFSClient
+
+lakefs_sdk_version = tuple(int(v) for v in __lakefs_sdk_version__.split("."))
```

Comment:
Is this even still referenced? (my VSCode doesn't find any references)

Reply:
No, but I figured it was worth keeping around if we have to do some version guarding in the future. I can also remove it though.